source/docs/articles/drupal/content-delivery-network-cdn-for-file-distribution.md

```diff
@@ -21,7 +21,6 @@ If you have streaming content, large amounts of multi-national traffic or need t
 
 If you don't meet those specific criteria, strongly consider just using Pantheon's existing infrastructure. It's highly optimized and can handle terrifying amounts of traffic. Many Enterprise clients with very high-profile sites do not use CDNs and rely on Pantheon's optimized stack to deliver the performance their customers demand.
 
-<div class="alert alert-info" role="alert">
```

Comment:
[Recent PR's](https://github.com/pantheon-systems/documentation/pull/539/files#diff-f7dcd7336c701c322f79814baa7306fbR25) include definition lists styled in the info alert class, if this isn't the desired standard I can remove them.